core/core.go

```diff
@@ -421,11 +420,40 @@ func (c *Core) AddArtifact(organization string, environment string, content stri
 	}
 
 	return client.InsertOrReplace(organizationDbConfiguration.Sql.Database, databaseConfig.Sql.Table, sqlElement.Name, sqlElement.Kind, string(sqlElementString))
+}
+
+func (c *Core) addArtifactToVault(organization string, environment string, content string) error {
+	var artifact models.Artifact
+	unmarshallError := json.Unmarshal([]byte(content), &artifact)
+	if unmarshallError != nil {
+		fmt.Printf("Unmarshalling error : %v\n", content)
+		fmt.Printf("Unmarshalling error : %v\n", unmarshallError.Error())
```

Comment:
You may want to change it to use logging package.

Reply:
done